content/en/workflows/service_accounts.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Service Accounts for Workflows
+kind: Documentation
+disable_toc: false
+further_reading:
+- link: "/account_management/org_settings/service_accounts/"
+  tag: "Documentation"
+  text: "Learn more about Datadog service accounts"
+---
+
+To add an automated or scheduled trigger to a workflow, the workflow must have an associated service account. The service account is needed to:
+- resolve the connections defined in the workflow actions at runtime
+- provide an identity for workflow executions
+- provide an identity for workflow audit trails
+
+To use automated or scheduled triggers, a workflow must be associated with a unique service account.
+
+## Requirements
+
+To create a service account for a workflow:
+- You must have either the Datadog admin role, or a custom role with the `SERVICE_ACCOUNT_WRITE` permission. For more information on service accounts and permissions, see [Service accounts][1].
+- Before you can add a service account, you need a workflow with at least one action.
+
+## Associate a service account with a workflow
+
+You can dynamically create a service account for your workflow when you [add a trigger][5]. Alternatively, click on the workflow name and add a service account next to **@worklow-** to dynamically create a service account and associate it with the workflow.
+
+{{< img src="workflows/create-service-account.png" alt="You can create a service account by clicking on the workflow name" style="width:70%;" >}}
+
+Service accounts use the email address associated with your Datadog account and follow the pattern: `[WF]:<your-workflow-name>`.
+
+## Service account roles and permissions
+
+When a workflow action runs, it checks the service account's roles to make sure that it is allowed to execute. For example, the **AWS Lambda** action requires the `CONNECTION_RESOLVE` resource permissions. The Datadog Admin Role and Datadog Standard Role both have `CONNECTION_RESOLVE` resource permissions, so the service account would require one of those roles before the **AWS Lambda** action could execute.
```

Comment:
```suggestion
When a workflow action runs, it checks the service account's roles to make sure that it is allowed to execute. For example, the **AWS Lambda** action requires the `CONNECTION_RESOLVE` resource permission. The Datadog Admin Role and Datadog Standard Role both have the `CONNECTION_RESOLVE` resource permission, so the service account requires one of those roles before the **AWS Lambda** action could execute.
```

Reply:
The `CONNECTION_RESOLVE` permission isn't visible in the UI. It's split from the Datadog Standard Role. (And admins also have it.) What you wrote is consistent with that. But is it confusing to refer to a permission that isn't visible in the UI? And would it be simpler to just say something like: workflow service accounts must have the Datadog Standard Role or Datadog Admin Role.

(I'm fine either way! Just want to make sure you're aware that "CONNECTION_RESOLVE" isn't visible anywhere in the UI.)